docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+node app @apostrophecms/asset:clear-cache
+```
+</AposCodeBlock>
+
+## Webpack configuration
+
+The webpack configuration can be extended from any module. The `asset` module exposes three different properties for accomplishing this.
+
+## Webpack options
+|  Property | Type | Description |
+|---|---|---|
+| [`extensions`](#extensions) | Object | Adds extra processing functionality to webpack. |
+| [`extensionOptions`](#extensionoptions) | Object | Adds functionality to any extensions added through the `extensions` option. |
+| [`bundles`](#bundles) | Object | |
+
+### `extensions`
+The `extensions` option allows the addition of custom methods for processing the Javascript and SCSS in the `ui/src` folder. The schema used within the option conforms to that used for configuring webpack, which you can learn more about [here](https://webpack.js.org/configuration/). This property can take either a simple object with a name property or a function that returns an object. 
+
+#### `extensions` simple object
+
+#### Example
+
+Modification of the `resolve` property of the webpack configuration using a simple object. New values will be merged with existing values:
+
+<AposCodeBlock>
+
+```javascript
+const path = require("path");
+
+module.exports = {
+ // ...
+ webpack: {
+   extensions: {
+     modifyResolve: {
+       resolve: {
+         extensions: [ '.jsx' ],
+         alias: {
+           'Utilities': path.join(process.cwd(), 'lib/utils')
+         }
+       }
+     }
+   }
+ },
+ // ...
+};
+
+```
+<template v-slot:caption>
+modules/my-module/index.js
+</template>
+</AposCodeBlock>
+
+::: note
+Make sure the new values are of the same type as the existing values, or they will replace the existing values, e.g., the webpack configuration `extensions` property takes an array, whereas `alias` takes an object.
+:::
+
+This will add both processing for files with an extension of `.jsx` and expose a shortcut to the `lib/utils` folder that we can use in our frontend files. For example, to import a custom method from our `methods.js` library in this folder.
+
+<AposCodeBlock>
+
+```javascript
+import  { customMethod } from ('Utilities/methods.js');
+
+export default () => {
+// Code utilizing customMethod
+};
+```
+<template v-slot:caption>
+modules/my-module/ui/src/index.js
+</template>
+</AposCodeBlock>
+
+Adding a unique name for the object within the apostrophe `extensions` property is important. Passing an object with the same name to an `extensions` property of another module will cause a conflict. The configuration file will be set to the `extensions` object in the last loaded module.
+
+#### `extensions` function
+
+Functions in the `extensions` property take an `option` parameter to allow multiple modules to modify the same extension through the [`extensionOptions`](#extensionoptions) property. Each property to be altered across multiple modules must be present in the option that was originally returned.
```

Comment:
option -> options

"can take"

"to allow several modules to pass options to an extension"

The last sentence is not true.

All we're saying here is that an extension can, if it wants to, be a function that takes an "options" object and return a function. And other modules can, if they want to, use a matching "extensionOptions" section to just merge in some additional properties of that object, or provide a function that takes that object so far, tweaks its properties and returns it.